pages/projects/new.tsx

```diff
@@ -20,6 +22,8 @@ const New: NextPage = () => {
         '';
 
     const redirectToProjectList = () => router.push('/projects');
+    const redirectToProjectDetails = (newProject: ApiUpdateResponse<Project>) =>
+        newProject.isSuccess && router.push(`/projects/${newProject.data.id}`);
```

Comment:
I'd prefer calling this `newProject` something like `projectCreate` instead, because the actual project is contained in the `data` field upon success.